Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Set explicit platform for ARM-based Macbooks
+FROM --platform=linux/amd64 python:3.10.13-slim-bookworm
+
+# Install dependencies in container
+RUN apt-get update && \
+    apt-get install -y nodejs graphviz && \
```

Comment:
You don't need these two lines.

Reply:
Removed! These were old lines that I just cleaned up!